man/rpm-ostree.xml

```diff
@@ -264,6 +265,11 @@ Boston, MA 02111-1307, USA.
             top of a new deployment.
           </para>
 
+          <para>When trying to uninstall a package that was installed
+            from a local RPM (i.e. not from a repository), you must
+            must specify the full NEVRA of the package.
```

Comment:
Must we have two musts?

Reply:
LOL...I knew I was going to miss something.  Still ENOTENOUGHCOFFEE here